packages/cms/src/member/MetaEditor.jsx

```diff
@@ -367,7 +370,7 @@ class MetaEditor extends Component {
    * Then call prepData to turn the sourceData into columns for the table.
    */
   hitDB() {
-    const searchGet = axios.get("/api/search/all");
+    const searchGet = axios.get("/api/search?q=&locale=all");
```

Comment:
does this mean that you can also completely delete the `"/api/search/all"` endpoint from `searchRoute.js`?